src/Timer/Program.cs

```diff
@@ -0,0 +1,63 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Client;
+using Temporalio.Worker;
+using TemporalioSamples.Timer;
+
+// Create a client to localhost on default namespace
+var client = await TemporalClient.ConnectAsync(new("localhost:7233")
+{
+    LoggerFactory = LoggerFactory.Create(builder =>
+        builder.
+            AddSimpleConsole(options => options.TimestampFormat = "[HH:mm:ss] ").
+            SetMinimumLevel(LogLevel.Information)),
+});
+
+async Task RunWorkerAsync()
```

Comment:
We haven't on most samples, but we may find using https://github.com/temporalio/sdk-dotnet/tree/main/src/Temporalio.Extensions.Hosting for worker running to be easier in our samples here and elsewhere (this is what many of our users use)

Reply:
Making worker file shorter sgtm, going to leave this for future work for time's sake.